src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -211,6 +216,8 @@ public final class RealJenkinsRule implements TestRule {
     private static final Pattern SNAPSHOT_INDEX_JELLY = Pattern.compile("(file:/.+/target)/classes/index.jelly");
 
     private final PrefixedOutputStream.Builder prefixedOutputStreamBuilder = PrefixedOutputStream.builder();
+    private boolean https;
+    private SSLContext sslContext;
```

Comment:
Could just make `sslContext` be `@CheckForNull` and omit `https` as implied by a non-null value.